core/tests/import_export_file_tests.rs

```diff
@@ -0,0 +1,134 @@
+mod integration_test;
+
+#[cfg(test)]
+mod import_export_file_tests {
+    use lockbook_core::model::state::temp_config;
+    use lockbook_core::service::import_export_service::ImportExportFileInfo;
+    use lockbook_core::service::test_utils::generate_account;
+    use lockbook_core::{
+        create_account, create_file, create_file_at_path, export_file, get_file_by_path, get_root,
+        import_file, write_document,
+    };
+    use lockbook_models::file_metadata::FileType;
+    use rand::Rng;
+    use uuid::Uuid;
+
+    #[test]
+    fn import_file_successfully() {
+        // new account
+        let config = &temp_config();
```

Comment:
Just take a reference when the variable is being passed somewhere (same in export test below).

Reply:
I did it this way cause a lot of other tests did as well, just to be consistent.

Reply:
Yea I think it's a good time to start doing it the right way. We'll go back later and clean up the rest, but just for this PR it we can start the change.

Reply:
down to not do this in all tests, @SmailBarkouch can you make an issue?

Reply:
Will update this file with fixes, and added [issue](https://github.com/lockbook/lockbook/issues/853).